manual/src/cmds/tsan.etex

```diff
@@ -0,0 +1,227 @@
+\chapter{Runtime detection of data races with ThreadSanitizer}
+\label{c:tsan}\cutname{tsan.html}
+%HEVEA\cutname{tsan.html}
+
+\section{s:tsan-overview}{Overview and usage}
+
+OCaml, since version 5.0, allows shared-memory parallelism and thus mutation of
+shared data. This creates the possibility of data races, i.e., unordered
+accesses to the same data (one of them being a write). Data races can be
+dangerous as they are easy to miss and capable of causing unexpected results.
+More information about data races and their consequences can be found in
+section~\ref{s:par_mm_easy} and Chapter~\ref{c:memorymodel}.
+
+To help detect data races, OCaml supports the ThreadSanitizer runtime detector
+since OCaml 5.2. To use ThreadSanitizer (also known as TSan), you must install
```

Comment:
"you must configure the compiler with \texttt{--enable-tsan}. You can also install an `opam` switch with the TSan feature enabled as follows:"